plugins/csv-import/src/csv.ts

```diff
@@ -97,66 +108,98 @@ const findRecordValue = (record: CSVRecord, key: string) => {
 const collator = new Intl.Collator("en", { sensitivity: "base" })
 const BOOLEAN_TRUTHY_VALUES = /1|y(?:es)?|true/iu
 
-function getRecordValueForField(
-    field: CollectionField,
+function getFieldInputForField(
+    field: Field,
     value: string | null,
     allItemIdBySlug: Map<string, Map<string, string>>
-) {
-    if (value === null) {
-        return undefined
-    }
-
+): FieldDataEntryInput | ConversionError {
     switch (field.type) {
         case "string":
+            if (value === null) {
+                return { type: "string", value: "" }
+            }
+            return { type: "string", value }
+
         case "formattedText":
-        case "link":
+            if (value === null) {
+                return { type: "formattedText", value: "" }
+            }
+            return { type: "formattedText", value }
+
         case "color":
+            return { type: "color", value }
+
+        case "link":
+            return { type: "link", value }
+
         case "file":
+            return { type: "file", value }
+
         case "image":
-            return value.trim()
+            return { type: "image", value }
```

Comment:
Should probably keep the `trim()`

```suggestion
            return { type: "image", value: value ? value.trim() : null }
```